client/js/components/day/staff-group.js

```diff
@@ -10,29 +11,57 @@ var StaffGroup = React.createClass({
   getDefaultProps: function(){
     return {
       approved: [],
-      needed: '',
-      name: ''
+      needed: Infinity,
+      name: '',
+      gigID: Infinity
     };
   },
 
   render: function(){
-    var approved = _.map(this.props.approved, function(human, idx){
-      return <Employee rating={human.rating} name={human.name} key={idx} />
+    var props = this.props;
+    var health = props.approved.length / props.needed;
+
+    var approved = _.map(props.approved, function(human, idx){
```

Comment:
Glad we are handling the human case, will make our app easily extendible for other non-humanoid life forms :+1: